<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>evaluateview</class>
 <widget class="QWidget" name="evaluateview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ratingLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>评价</string>
   </property>
  </widget>
  <widget class="QSlider" name="ratingSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提交评价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您的评价对我们很重要！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请给出您对司机的评价:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
